### DISCUSSION TITLE
What to do about unmaintained Emacs packages

### DISCUSSION THREAD

**Discussion by niontrix** [syl20bnr/spacemacs#16246] [General]
Hi there,

I recently created a pull request to fix an issue in `flycheck-nim` that prevents it from working. I did not receive any feedback from the maintainer of the Github repo, neither did he merge my pull request. Taking from his activity chart on Github he seems to be inactive for an extended period of time. So, I wrote him an email kindly asking if he could take a look at my pull request and maybe merge it in. Unfortunately I still haven't received any answer.
My question now is: What can I do about that issue? Also, because there is a MELPA package with the same issue that won't get updated. Could we somehow replace it? Or would people need to use a custom recipe that would link to my fork, for example? (Which would be fairly cumbersome)

Any ideas?

KR
Tobias

**Comment by alexey0308** [comment]
Hi Tobias, you may suggest to the formal maintainers to create a fork under their user (see for example https://github.com/syl20bnr/spacemacs/issues/15541#issuecomment-1140458990).

**Comment by niontrix** [comment]
@alexey0308 Thanks for your suggestion. Do you know how I can find the formal maintainer of the `flycheck-nim` package?

**Comment by alexey0308** [comment]
> @alexey0308 Thanks for your suggestion. Do you know how I can find the formal maintainer of the `flycheck-nim` package?

Hi @niontrix, I meant to contact the maintainers of spacemacs, for example @smile13241324 (?).

**Reply by smile13241324** [reply]
Hi guys,

thanks for the report, would you consider making a ticket for this? I would then have a look.

**Comment by smile13241324** [comment]
@niontrix I have forked the package, please make a PR against my fork and I will merge it. The version of the package has already been increased to 0.2.1 so that the changes should be installed with the next regular package update.

In future I would like to receive a ticket for such a broken ticket. There we should discuss if there is a maintained alternative we could use or even drop the dependency. If all of this is not feasible I can make a fork and accept PRs for this. Still we should try to get such changes back to melpa i.e. into the orphanage. It would really be a great help if some people would jump in for maintaining such old emacs packages and apply to melpa as new maintainers.

**Comment by niontrix** [comment]
@smile13241324 Thanks for creating the fork! I'll create a PR against it and think about applying as a new melpa maintainer.
